src/wp-includes/class-wp-theme.php

```diff
@@ -824,6 +824,38 @@ public function cache_delete() {
 		$this->delete_pattern_cache();
 	}
 
+	/**
+	 * Get block pattern cache.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @return mixed Returns an array of patterns if cache is found, otherwise false.
```

Comment:
Maybe clarify the return type?
```suggestion
	 * @return array|false Returns an array of patterns if cache is found, otherwise false.
```
(or `array|bool` if you prefer that)